.editorconfig

```diff
@@ -1,5 +1,5 @@
 root = true
 
-[*.js]
+[*.{js,json}]
```

Comment:
1d4869a

Technically, we could have a separate rule for `./package.json` and the rest.